Microsoft.Toolkit.Uwp.UI.Controls.Layout/ListDetailsView/ListDetailsView.cs

```diff
@@ -47,7 +47,7 @@ public partial class ListDetailsView : ItemsControl
         private ContentPresenter _detailsPresenter;
         private VisualStateGroup _selectionStateGroup;
         private Button _inlineBackButton;
-        private object _navigationView;
+        private Microsoft.UI.Xaml.Controls.NavigationView _navigationView;
```

Comment:
We could add a using at the top as well so we don't have to copy the namespace around:
```
using NavigationView = Microsoft.UI.Xaml.Controls.NavigationView;
```